tests/utils/managed_process.py

```
@@ -0,0 +1,242 @@
+        for url in self.health_check_urls:
+            time_taken += self._check_url(url, timeout)
+        return time_taken
+
+    def _check_url(self, url, timeout=30, sleep=0.1):
+        if isinstance(url, tuple):
+            response_check = url[1]
+            url = url[0]
+        else:
+            response_check = None
+        start_time = time.time()
+        self._logger.info(f"Checking URL {url}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)